libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -63,9 +63,18 @@ const ftx: SwapProviderConfig = {
   }),
   needsKYC: true,
   needsBearerToken: true,
+  type: "CEX",
 };
 
-const swapProviders: Record<string, SwapProviderConfig> = {
+type CEXProviderConfig = SwapProviderConfig & { type: "CEX" };
+type DEXProviderConfig = {
+  needsKYC: boolean;
```

Comment:
Check if we need these @cjordan-ledger

Reply:
I think we do need these properties on DEX providers as well but they can be set to false. It makes the code simpler if all providers have `needsKYC`/`needsBearerToken`